CHANGELOG.md

```diff
@@ -108,7 +110,8 @@ the statistics service.
 -   Using headless Firefox to run tests for CI
 -   Using node 10.13.0
 
-[Unreleased]: https://github.com/ACWI-SOGW/ngwmn-ui/compare/ngwmn-ui-0.10.0...master
+[Unreleased]: https://github.com/ACWI-SOGW/ngwmn-ui/compare/ngwmn-ui-0.11.0...master
```

Comment:
This should be ngwmn-ui-0.10.0 ... master

Reply:
Ah, thank you!